syslog-ng-dev/dev-dependencies.txt

```diff
@@ -33,4 +33,5 @@ xsltproc
 # required for make
```

Comment:
I'd prefer a commit message more like this (or split to multiple lines more verbosely):
`dev-dependencies.txt: add libxml2-utils (dbparser/tests/test_patterndb)`